### ISSUE TITLE
"bash: -o: command not found" when recompiling libaccessibility_toolkit_s.a.desc

### ISSUE THREAD

**Issue by jlebar** [bgirard/hacky.mk#4]
I touched mfbt/LinkedList.h, ran reninja, then ran ninja, and got

```
[38/2916] accessible/src/atk/libaccessibility_toolkit_s.a.desc FAILED: bash
-c "cd accessible/src/atk && rm -f libaccessibility_toolkit_s.a
../libaccessibility_toolkit_s.a &&  -o libaccessibility_toolkit_s.a.desc
AccessibleWrap.o ApplicationAccessibleWrap.o AtkSocketAccessible.o
DocAccessibleWrap.o Platform.o RootAccessibleWrap.o UtilInterface.o
nsMaiHyperlink.o nsMaiInterfaceAction.o nsMaiInterfaceComponent.o
nsMaiInterfaceDocument.o nsMaiInterfaceEditableText.o
nsMaiInterfaceHyperlinkImpl.o nsMaiInterfaceHypertext.o
nsMaiInterfaceImage.o nsMaiInterfaceSelection.o nsMaiInterfaceTable.o
nsMaiInterfaceText.o nsMaiInterfaceValue.o" bash: -o: command not found
```

**Comment by vvuk** [comment]
Fixed in 585ebb876398194c -- the rules.mk patch was busted.  Also note that with this commit, reninja is gone -- run 'python ../path/to/config/hackymake/hackypost.py'
